lint-webos/webos-result.html

```diff
@@ -0,0 +1,427 @@
+<!DOCTYPE html>
```

Comment:
Should this file be checked in? Shouldn't this be generated each time as a demo to the user?

Reply:
it shouldn't be here. I just added it temporarily to show what the HTML output file looks like at thie moment, and I prefer the summary to come first and then the results list. I'm assuming the results list is very large as the webOS file is very large.